nvflare/lighter/impl/master_template.yml

```diff
@@ -533,4 +533,139 @@ dockerfile: |
   RUN pip install -U pip
   RUN pip install nvflare
   COPY requirements.txt requirements.txt
-  RUN pip install -r requirements.txt
\ No newline at end of file
+  RUN pip install -r requirements.txt
+
+helm_chart_chart: |
+  apiVersion: v2
+  name: nvflare
+  description: A Helm chart for NVFlare overseer and servers
+  type: application
+  version: 0.1.0
+  appVersion: "2.2.0"
+
+helm_chart_service_overseer: |
+  apiVersion: v1
+  kind: Service
+  metadata:
+    name: overseer
+  spec:
+    selector:
+      system: overseer
+    ports:
+      - protocol: TCP
+        port: 8443
+        targetPort: overseer-port
+
+helm_chart_service_server: |
+  apiVersion: v1
+  kind: Service
+  metadata:
+    name: server
+    labels:
+      system: server
+  spec:
+    selector:
+      system: server
+    ports:
+      - name: fl-port
+        protocol: TCP
+        port: 8002
+        targetPort: fl-port
+      - name: admin-port
+        protocol: TCP
+        port: 8003
+        targetPort: admin-port
+
+helm_chart_deployment_overseer: |
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: overseer
+    labels:
+      system: overseer
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        system: overseer
+    template:
+      metadata:
+        labels:
+          system: overseer
+      spec:
+        volumes:
+          - name: workspace
+            hostPath:
+              path:
+              type: Directory
+        containers:
+          - name: overseer
+            image: nvflare-min:2.2.0
```

Comment:
how do we get this image build, and this name?

Reply:
This will be replaced by the docker_image in helm chart builder.  Basically, anything running in k8s cluster must be docker images.  The person who controls the k8s, with kubectl or helm, knows where to pull the images and he needs to build the image and pushes it to the registry.

Reply:
got it, we will need some documentation updates to explain these stuff to the users.